lib/reports/matched_content_changes_report.rb

```diff
@@ -0,0 +1,32 @@
+class Reports::MatchedContentChangesReport
+  OUTPUT_ATTRIBUTES = {
+    created_at: :content_change,
+    base_path: :content_change,
+    change_note: :content_change,
+    document_type: :content_change,
+    publishing_app: :content_change,
+    priority: :content_change,
+    title: :subscriber_list,
+    slug: :subscriber_list,
+  }.freeze
+
+  def call(start_time: nil, end_time: nil)
+    start_time = start_time ? Time.zone.parse(start_time) : 1.week.ago
+    end_time = end_time ? Time.zone.parse(end_time) : Time.zone.now
+
+    query = MatchedContentChange
+      .includes(:subscriber_list, :content_change) # for efficient "row.content_change"
+      .joins(:content_change) # for the next query
+      .where("content_changes.created_at": start_time..end_time)
```

Comment:
It'd be good to have an ordering on this, otherwise the report might be rather surprising with same content change spread out across the results.

Reply:
Sure, I've added one.